.github/workflows/ci_tests.yml

```diff
@@ -31,13 +31,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ubuntu-latest]
+        platform: [ubuntu-24.04]
         compiler:
           - cpp: g++
             c: gcc
           - cpp: clang++
             c: clang
-        cpp_version: [17, 20, 23, 26]
```

Comment:
Can we not keep this with the new ref impl? It requires C++23?

Reply:
[from_range](https://en.cppreference.com/w/cpp/ranges/from_range), e.g., is C++23.
I'd suggest bringing the ref implementation to main first and then explore if and how earlier standards can be enabled.